Dynatrace/dynatrace-operator
# Extend support archive by managed operator components

## Description

This PR should extend the support archive to not only collect logs and manifests for rolled out operator components, but also for components that are rolled out by the operator itself.

In addition a new command line flag was introduced (`--managed-logs=false`) to stop collecting logs for rolled out components.

## How can this be tested?

Run the support archive and check for new files: 
```
kubectl exec -n dynatrace deployment/dynatrace-operator -- dynatrace-operator support-archive --stdout > operator-support-archive.tgz
```

## Checklist
